.circleci/config.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+jobs:
+   build:
+     docker:
+       - image: circleci/python:3.5
+     steps:
+       - checkout
+       - restore_cache:
+           key: deps1-{{ .Branch }}-{{ checksum "setup.py" }}
+       - run:
+           command: |
+             python3 -m venv venv
+             . venv/bin/activate
+       - run: sudo apt-get install -y libgmp3-dev
```

Comment:
would do this as first command, to split between
- install global dependencies
- setup venv
- run